renku_notebooks/api/classes/server.py

```diff
@@ -314,6 +314,9 @@ def _get_session_manifest(self):
             current_app.config["RENKU_ANNOTATION_PREFIX"]
             + "projectName": self.gl_project.path.lower(),
             f"{current_app.config['RENKU_ANNOTATION_PREFIX']}requested-image": self.image,
+            f"{current_app.config['RENKU_ANNOTATION_PREFIX']}repository": None
+            if self.gl_project is None
```

Comment:
There's a couple of places above this line, e.g. `current_app.config["RENKU_ANNOTATION_PREFIX"] + "projectName": self.gl_project.path.lower()` where we also access properties of `gl_project` which could be None, in which case the code would fail. I think this should be handled on those lines, too.